helios-testing/src/main/java/com/spotify/helios/testing/HeliosSoloDeployment.java

```diff
@@ -157,6 +161,31 @@
     }
   }
 
+  /**
+   * Return first found non-loopback ip from enumeration of network devices.
+   * InetAddress.getLocalHost() will not work, as it returns the loopback device
+   */
+  public static String findFirstInet4Addr(Enumeration<NetworkInterface> nics)
+      throws HeliosDeploymentException {
+    List addrs = new ArrayList<InterfaceAddress>();
+    for (NetworkInterface nic : Collections.list(nics)) {
+      addrs.addAll(nic.getInterfaceAddresses());
+    }
+    return findFirstInet4Addr(addrs);
+  }
+
+  public static String findFirstInet4Addr(Collection<InterfaceAddress> addrs)
+      throws HeliosDeploymentException {
+    for(InterfaceAddress addr : addrs) {
+      String addrStr = addr.getAddress().toString();
```

Comment:
also I think you can just do `InetAddress.getHostAddress()` to get the ip address as a string, rather than `toString`, so you won't have to bother with the `/` as the host-ip separator.